editor/src/DocumentWindows/MaterialInspector.cpp

```
@@ -0,0 +1,102 @@
+namespace nexo::editor {
+	void MaterialInspector::setup()
+	{
+		renderer::FramebufferSpecs framebufferSpecs;
+		framebufferSpecs.attachments = {renderer::FrameBufferTextureFormats::RGBA8, renderer::FrameBufferTextureFormats::Depth};
+		framebufferSpecs.width = static_cast<unsigned int>(64);
+		framebufferSpecs.height = static_cast<unsigned int>(64);
+		m_framebuffer = renderer::Framebuffer::create(framebufferSpecs);
+		m_framebuffer->setClearColor({0.05f, 0.05f, 0.05f, 0.0f});
+	}
+
+	void MaterialInspector::shutdown()
```

Cppcheck (2.10-2)

[style] The function 'shutdown' is never used.

(unusedFunction)